features/step_definitions/viewing_browse_steps.rb

```diff
@@ -1,5 +1,4 @@
-Given(/^there is a browse page set up with links$/) do
-
+Given(/^there is a (\w+) browse page set up with links$/) do |second_level_ordering|
```

Comment:
If you made this `/^there is an? (\w+)...` then you wouldn't have to have the incorrect grammar in the features (ie you could do "Given there is an alphabetical browse page...").